### ISSUE TITLE
Home page

### ISSUE THREAD

**Issue by EmilyScripts** [harryyy27/Brickworks-Tollington_Project#3]
- [ ] Header and footer sizing needs to be appropriate for screen size
- [ ] Reduce Brickworks image size/use different design (just the B maybe?)
- [ ] Change text font and sizing needs to be reduced
- [ ] Button needs styling so it doesn't look like its from 2001
- [ ] Spacing on home screen on desktop needs addressing

**Comment by EmilyScripts** [comment]
![selection_005](https://user-images.githubusercontent.com/32196496/49660133-c895a700-fa3d-11e8-821d-18572deb3319.png)

Did you want to simplify the logo so then we can do more with it/edit it to what we want as the other one and its text end up looking very pixelated and not very sharp

**Comment by harryyy27** [comment]
I suppose that can be one of the last things we look at. I wouldn't mind learning how to manipulate SVGs

**Comment by EmilyScripts** [comment]
Yeah definitely not top priority, the one on the navbar needs fixing too eventually, I did a hack job just for the meantime so we know the layout

**Comment by harryyy27** [comment]
also priority number 1, turn navbar into a canvas ball pit
